cmd/snap/cmd_services.go

```diff
@@ -188,7 +188,7 @@ func (s *svcLogs) Execute(args []string) error {
 type userAndScopeMixin struct {
 	System bool   `long:"system"`
 	User   bool   `long:"user"`
-	Users  string `long:"users" optional:"yes" optional-value:"all"`
+	Users  string `long:"users" optional:"yes"`
```

Comment:
this needs changes to the spread tests no? they were using --users

Reply:
Updated test to pass with the changes!